### DISCUSSION TITLE
Next Server error

### DISCUSSION THREAD

**Discussion by MOAIZ-UL-ISLAM** [vercel/next.js#54664] [Help]
### Summary

![image](https://github.com/vercel/next.js/assets/87712231/a854bf21-5fb5-437b-8503-c582e0db83f4)

### Additional information

```js
my server get crash after several minutes why is that so
```

**Comment by hyjoong** [comment]
I noticed that you encountered a 'Next Server error' when building with Next.js, and your server crashes after several minutes. Can you provide more details about this issue? Specifically, could you share the code where you're making the fetch request?

**Reply by MOAIZ-UL-ISLAM** [reply]
i'nt making any fetch requests for now however i am using different packages like framer-motion and other thing and my server crahes with these errors
